src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * 40 - 20;
```

Comment:
Replace the hardcoded values '3', '7', '20' with named constants to make the code more understandable and maintainable.